src/main/target/common_post.h

```diff
@@ -25,3 +25,11 @@
 # undef VTX_SMARTAUDIO
 # undef VTX_TRAMP
 #endif
+
+#if defined(SPEKTRUM_BIND_PIN) && !defined(SPEKTRUM_BIND)
+#define SPEKTRUM_BIND
+#endif
+
+#if !defined(SPEKTRUM_BIND_PIN) && defined(SPEKTRUM_BIND)
```

Comment:
i think it probably is better, to just have `SPEKTRUM_BIND_PIN`, if only avoid further confusion.  you can't do spektrum binding without a bind pin so it kinda of make sense.  I'm fine with either solution however, at least then the issue doesn't spread to all the target files.